### PR TITLE
Update v2 dashboard

### PR DESCRIPTION
**Pull request by agnivade**
And also remove the v1 dashboard as I removed it from grafana.com.
There's no need to use that anymore.

### CODE REVIEW

**Review by agarciamontoro — APPROVED** [review]
Tested this importing the JSON to a Grafana instance in one of the load-test clusters and it worked! Thanks :)

Speaking of which: should we add it to the load-test's dashboards as well? :thinking:

**Comment by agnivade** [comment]
Yeah good point. I guess we should.

File: grafana/mattermost-performance-monitoring-v2.json [hunk 1]
```diff
@@ -2608,12 +2858,13 @@
     ]
   },
   "time": {
-    "from": "now-6h",
```
Comment (streamer45) [inline grafana/mattermost-performance-monitoring-v2.json]: Not a big deal but just want to confirm this is wanted and not just a result of exporting the current state.

File: grafana/mattermost-performance-monitoring-v2.json [hunk 2]
```diff
@@ -2608,12 +2858,13 @@
...
   },
   "timepicker": {},
   "timezone": "",
   "title": "Mattermost Performance Monitoring v2",
-  "uid": "im7xNX17k",
-  "version": 4
-}
+  "uid": "im7xNX17kdd",
+  "version": 1,
+  "weekStart": ""
```
Comment (streamer45) [inline grafana/mattermost-performance-monitoring-v2.json]: Should the ID change? I'd only expect a version bump.

File: grafana/mattermost-performance-monitoring-v2.json [hunk 3]
```diff
@@ -2455,58 +2684,63 @@
...
+          "refId": "A"
+        }
+      ],
       "title": "System Metrics",
       "type": "row"
     }
   ],
-  "refresh": "1m",
-  "schemaVersion": 26,
+  "refresh": "5s",
```
Comment (streamer45) [inline grafana/mattermost-performance-monitoring-v2.json]: Similar concern, `5s` refresh may be slightly aggressive as a default but will leave it up to you.

**Review by streamer45 — APPROVED** [review]
Nothing blocking, just a couple of comments.

File: grafana/mattermost-performance-monitoring-v2.json (hunk 1 above)
Comment (agnivade) [reply grafana/mattermost-performance-monitoring-v2.json]: This wasn't intentional

File: grafana/mattermost-performance-monitoring-v2.json (hunk 2 above)
Comment (agnivade) [reply grafana/mattermost-performance-monitoring-v2.json]: The id can change, that's fine.

File: grafana/mattermost-performance-monitoring-v2.json (hunk 3 above)
Comment (agnivade) [reply grafana/mattermost-performance-monitoring-v2.json]: Let me change that.

**Pull request merged**